examples/jsm/offscreen/offscreen.js

```diff
@@ -1,8 +1,12 @@
-import init from './scene.js';
+import { init } from './scene.js';
 
-self.onmessage = function ( message ) {
+/* @__PURE__ */ ( () => {
 
-	const data = message.data;
-	init( data.drawingSurface, data.width, data.height, data.pixelRatio, data.path );
+	self.onmessage = function ( message ) {
 
-};
+		const data = message.data;
+		init( data.drawingSurface, data.width, data.height, data.pixelRatio, data.path );
+
+	};
+
+} );
```

Comment:
This is where CI is failing since the expression isn't referenced. Is offscreen used for anything though?